freqtrade/tests/conftest.py

```diff
@@ -51,13 +51,13 @@ def get_patched_freqtradebot(mocker, config) -> FreqtradeBot:
     """
     # mocker.patch('freqtrade.fiat_convert.Market', {'price_usd': 12345.0})
     patch_coinmarketcap(mocker, {'price_usd': 12345.0})
-    mocker.patch('freqtrade.freqtradebot.Analyze', MagicMock())
+#    mocker.patch('freqtrade.freqtradebot.Analyze', MagicMock())
```

Comment:
i guess we should remove those if the class does not exist anymore...

Reply:
Fixed in 85fd4dd